client/components/modals/podcast/NewModal.vue

```diff
@@ -205,9 +216,13 @@ export default {
       this.podcast.itunesPageUrl = this._podcastData.pageUrl || ''
       this.podcast.itunesId = this._podcastData.id || ''
       this.podcast.itunesArtistId = this._podcastData.artistId || ''
-      this.podcast.language = this._podcastData.language || ''
+      this.podcast.language = this._podcastData.language || this.feedMetadata.language || ''
       this.podcast.autoDownloadEpisodes = false
 
+      if (this._podcastData.explicit === 'yes' || this._podcastData.explicit == 'true' || this.feedMetadata.explicit === 'yes' || this.feedMetadata.explicit == 'true') {
+        this.podcast.explicit = true
+      } else this.podcast.explicit = false
+
```

Comment:
```suggestion
      this.podcast.explicit = this._podcastData.explicit || this.feedMetadata.explicit === 'yes' || this.feedMetadata.explicit == 'true'
```

`this._podcastData.explicit` should always be a Boolean since we are parsing that in the provider.